Cargo.toml

```diff
@@ -44,7 +44,10 @@ async-trait = { version = "0.1" }
 bytes = { version = "1.5" }
 chrono = { version = "0.4.32" }
 cloudevents-sdk = { version = "0.7", optional = true }
+env_logger = { version = "0.11.3" }
+log = { version = "0.4" }
```

Comment:
I might have missed something - but in the PR, the logger is only used within uuidbuilder tests? Shouldn't this be a dev-dependency then? Also, do we really need to pull this in, compared to what we're actually using it for? 

If we have to do logging, nothing at all against doing it properly. But so far we always got away without...